File: factory-templates/node/node-typescript-cli/src/commands/menu.command.ts
import { CommandLineInput } from "../models/command-line-input";
import { ICommand } from "./command";

export class MenuCommand implements ICommand {

  get name(): string { return "menu"; }

  run = (args: CommandLineInput): Promise<void> => {
    return new Promise(res => {
      console.log("\r\nPlease choose from one of the following options:");
      console.log("Type 'menu' to display this menu.");
      console.log("Type 'echo' to open echo cli.");
      console.log("Type 'exit' to close CLI.\r\n");
      res();
    })
  }

}

File: factory-templates/dotnet/dotnet-csharp-cli/Commands/MenuCommand.cs
using System;
using System.Threading.Tasks;

namespace Shaman.CommandLineUtility.Commands;

public class MenuCommand : ICommand
{
    public string Name => "menu";

    public Task<bool> Run()
    {
        Console.WriteLine($"{Environment.NewLine}Please choose from one of the following options:");
        Console.WriteLine("Type 'menu' to display this menu.");
        Console.WriteLine("Type 'echo' to open echo cli.");
        Console.WriteLine($"Type 'exit' to close CLI.{Environment.NewLine}");
        return Task.FromResult(true);
    }
}